README.md

```diff
@@ -38,8 +38,17 @@ A multiple-thread based Linux network throughput benchmark tool.
 
 ### Building NTTTCP-for-Linux ###
 
+- Using `make`:
+
 	make; make install
 
+- Using `CMake`:
+
+	cd src
+	mkdir build && cd build
+	cmake ..
+	make && make install
```

Comment:
I don't think that this actually works. It doesn't look like CMake was setup with the `install` rule to specify which target should be installed where.